improver_tests/ensemble_copula_coupling/ensemble_copula_coupling/test_utilities.py

```diff
@@ -553,5 +556,85 @@ def test_missing_coordinate(self):
                 cube.data, cube, "nonsense", plen)
 
 
+class Test_merge_land_and_sea(IrisTest):
+
+    """Test merge_land_and_sea"""
+
+    def setUp(self):
+        """Set up a percentile cube"""
+        # Create a percentile cube
+        land_data = np.ones((2, 3, 4), dtype=np.float32)
+        sea_data = np.ones((2, 3, 4), dtype=np.float32)*3.0
+        mask = np.array([[[True, False, False, False],
+                          [True, False, False, False],
+                          [False, False, False, True]],
+                         [[True, False, False, False],
+                          [True, False, False, False],
+                          [False, False, False, True]]])
+        land_data = np.ma.MaskedArray(land_data, mask)
+        self.percentiles_land = set_up_percentile_cube(land_data, [30, 60])
+        self.percentiles_sea = set_up_percentile_cube(sea_data, [30, 60])
+
+    def test_missing_dim(self):
+        """Check that an error is raised if missing dimensional coordinate"""
+        single_percentile = squeeze(self.percentiles_land[0])
+        message = "Input cubes do not have the same dimension coordinates"
+        with self.assertRaisesRegex(ValueError, message):
+            merge_land_and_sea(single_percentile, self.percentiles_sea)
+
+    def test_mismatch_dim_length(self):
+        """Check an error is raised if a dim coord has a different length"""
+        land_slice = self.percentiles_land[:, 1:, :]
+        message = "Input cubes do not have the same dimension coordinates"
+        with self.assertRaisesRegex(ValueError, message):
+            merge_land_and_sea(land_slice, self.percentiles_sea)
+
+    def test_merge(self):
+        """Test merged data."""
+        expected_merged_data = np.array(
+            [[[3.0, 1.0, 1.0, 1.0],
+              [3.0, 1.0, 1.0, 1.0],
+              [1.0, 1.0, 1.0, 3.0]],
+             [[3.0, 1.0, 1.0, 1.0],
+              [3.0, 1.0, 1.0, 1.0],
+              [1.0, 1.0, 1.0, 3.0]]], dtype=np.float32)
+        expected_cube = self.percentiles_land.copy()
+        expected_cube.data = expected_merged_data
+        merge_land_and_sea(self.percentiles_land, self.percentiles_sea)
+        self.assertArrayAlmostEqual(
```

Comment:
I would have thought that ```numpy.testing.assert_array_equal``` would work here. There is nothing approximate about this, we haven't modified the values at all.